python/tskit/trees.py

```diff
@@ -1017,16 +1017,16 @@ def get_mrca(self, u, v):
         # Deprecated alias for mrca
         return self.mrca(u, v)
 
-    def mrca(self, u, v):
+    def mrca(self, *args):
         """
         Returns the most recent common ancestor of the specified nodes.
-
-        :param int u: The first node.
-        :param int v: The second node.
-        :return: The most recent common ancestor of u and v.
+        :param int args: input node IDs.
```

Comment:
I'm not sure what the right way to make a docstring for variable args input like this - @benjeffery, any ideas?

Reply:
Good catch, this isn't rendering correctly for me at least. I think this is the best you can do:
```suggestion

        :param int `*args`: input node IDs, must be at least 2.
```